src/translations/en.json

```diff
@@ -810,7 +810,8 @@
           "area": "Set entity area only",
           "area_note": "By default the entities of a device are in the same area as the device. If you change the area of this entity, it will no longer follow the area of the device.",
           "follow_device_area": "Follow device area",
-          "change_device_area": "Change device area"
+          "change_device_area": "Change device area",
+          "configure_state": "Configure State"
```

Comment:
Edit Options?

I think the word `edit` is better than configure for these. State is weird as you aren't changing the state. you could change the options of a select. which doesn't change the state

Reply:
Right. So configure state doesn't make sense for everything. The dialog is also called options.

Reply:
The dialog is called settings:

![image](https://user-images.githubusercontent.com/1444314/159538350-37963e83-c930-43fc-a35e-17e5bd984376.png)

Reply:
The dialog that configure state opens is that called options like it was in your edemo the other day?